product/stream_processor/integrations/events/providers/eventbridge.py

```diff
@@ -0,0 +1,87 @@
+import os
+from typing import TYPE_CHECKING, Generator, Optional
+
+import boto3
+import botocore.exceptions
+
+from product.constants import XRAY_TRACE_ID_ENV
+from product.stream_processor.integrations.events.base import EventProvider
+from product.stream_processor.integrations.events.constants import EVENTBRIDGE_PROVIDER_MAX_EVENTS_ENTRY
+from product.stream_processor.integrations.events.exceptions import ProductNotificationDeliveryError
+from product.stream_processor.integrations.events.functions import chunk_from_list
+from product.stream_processor.integrations.events.models.input import Event
+from product.stream_processor.integrations.events.models.output import EventReceipt, EventReceiptFail, EventReceiptSuccess
+
+if TYPE_CHECKING:
```

Comment:
not sure we really need it, we use lambda layer and pack everything anyways always

Reply:
that's because we shouldn't be including dev dependencies in the layer, unless I misunderstood and you're purposedfully including `mypy-boto3*` into the Layer (they can be bigger than our entire code!)